src/components/ProposalStatusForm.tsx

```diff
@@ -67,7 +74,8 @@ export default function ProposalStatusForm({
   } else if (
     (proposalDetails?.typeKey === 'STUDENT' &&
       proposalDetails?.statusKey === 'MATCHED') ||
-    proposalDetails?.receivedFeedbacks?.length > 0
+    proposalDetails?.receivedFeedbacks?.length > 0 ||
+    providedFeedback
   ) {
     return (
       <div className="p-4 bg-yellow-100">
```

Comment:
maybe add a check here if the providedFeedback is ACCEPT, also display the accept message?

Reply:
Done